src/main/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -53,6 +59,8 @@ export class ClaimantResponseConverter {
       return false
     } else if (draftClaimantResponse.intentionToProceed && draftClaimantResponse.intentionToProceed.proceed.option === YesNoOption.YES) {
       return false
+    } else if (draftClaimantResponse.intentionToProceed && draftClaimantResponse.intentionToProceed.proceed.option === YesNoOption.YES) {
+      return false
```

Comment:
lines 60, 61 and 62, 63 are same. Please check...

Reply:
Fixed